next.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   env: {
     WALLORA_BACKEND_SUBMIT_FEEDBACK: 'https://wt-5c9a92f46a5c0d255deef1b49b7375ca-0.run.webtask.io/wallora-feedback',
+    WALLORA_BACKEND_TOTAL_PLANNED_AMOUNT_WORKITEMWISE:
```

Comment:
will not need this anymore if we are using GQL

Reply:
keeping it still so that we might remove it when these apis are migrated to graphql.